python/kvikio/kvikio/_version.py

```diff
@@ -17,4 +17,4 @@
 __version__ = (
     importlib.resources.files("kvikio").joinpath("VERSION").read_text().strip()
 )
-__git_commit__ = ""
+__git_commit__ = "7b0231c8241164c8a0511cd737e7098cb7ccea3e"
```

Comment:
This change shouldn't be committed. I assume this got modified during a local CI run, same as the VERSION?
```suggestion
__git_commit__ = ""
```